lib/manager/terraform/readme.md

```diff
@@ -1,19 +1,36 @@
-Currently Terraform support is limited to Terraform registry sources and github sources that include SemVer refs, e.g. like `github.com/hashicorp/example?ref=v1.0.0`.
+Currently, Terraform supports renovating the following dependencies, where sub points represent hosting options of the dependencies:
 
-Fixed versions like the following will receive a PR whenever there is a newer version available:
+- modules
+  - GitTags
+  - GithubTags
+  - TerraformRegistry ( Public and Private )
+- providers ( deprecated in Terraform 0.13.0 )
+  - TerraformRegistry ( Public and Private )
+- required_providers block ( Terraform >= 0.13.0)
+  - TerraformRegistry ( Public and Private )
+- required_version
+- helm_release
+  - chart repository ( Public and Private )
+- docker\_\*
+  - Docker registry ( Public and Private )
 
-```
-module "consul" {
-  source  = "hashicorp/consul/aws"
-  version = "0.0.5"
-  servers = 3
-}
-```
-
-The following _range_ constraints are also supported:
+Terraform range constraints are supported:
 
 - `>= 1.2.0`: version 1.2.0 or newer
 - `<= 1.2.0`: version 1.2.0 or older
 - `~> 1.2.0`: any non-beta version >= 1.2.0 and < 1.3.0, e.g. 1.2.X
 - `~> 1.2`: any non-beta version >= 1.2.0 and < 2.0.0, e.g. 1.X.Y
-- `>= 1.0.0`, <= 2.0.0`: any version between 1.0.0 and 2.0.0 inclusive
+- `>= 1.0.0`, `<= 2.0.0`: any version between 1.0.0 and 2.0.0 inclusive
```

Comment:
```suggestion
- `>= 1.0.0, <= 2.0.0`: any version between 1.0.0 and 2.0.0 inclusive
```

I'm not sure if my previous suggestion was correct, maybe it should be this instead???

`>= 1.0.0, <= 2.0.0`